docs/source/providers.rst

```diff
@@ -179,3 +179,112 @@ Other Notes
+
+.. _`provider options`: https://github.com/pradels/vagrant-libvirt#provider-options
+.. _`libvirt connection URI`: http://libvirt.org/uri.html
+
+Here is an example of how could look like your molecule.yml file:
+
+.. code-block:: yaml
+
+Domain Specific Options
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* `disk_bus` - The type of `disk device`_ to emulate. Defaults to virtio if not set.
+* `nic_model_type` - parameter specifies the model of the network adapter when you create a domain value by default virtio KVM believe possible values, see the `nics documentation`_.
+* `memory` - Amount of memory in MBytes. Defaults to 512 if not set.
+* `cpus` - Number of virtual cpus. Defaults to 2 if not set.
+* `nested` - `Enable nested virtualization`_. Default is false.
+* `cpu_mode` - `CPU emulation mode`_. Defaults to 'host-model' if not set. Allowed values: host-model, host-passthrough.
+* `Other options` - Such as graphics_port, suspend_mode, boot, etc. Please, take a look at the `vagrant-libvirt`_ documentation for seeing all available options.
+
+.. _`disk device`: http://libvirt.org/formatdomain.html#elementsDisks
+.. _`nics documentation`: https://libvirt.org/formatdomain.html#elementsNICSModel
+.. _`Enable nested virtualization`: https://github.com/torvalds/linux/blob/master/Documentation/virtual/kvm/nested-vmx.txt
+.. _`CPU emulation mode`: https://libvirt.org/formatdomain.html#elementsCPU
+
+Usage
+^^^^^
+
+All libvirt specific options(such as the one above, provider specific and domain options) must be specified in the providers section. Nevertheless, other options such as synced or network settings should be added to the raw_config_args, as they are vagrant generic parameters. Note that you can use special libvirt parameters such as "libvirt__tunnel_type", as it is shown in the example below.
+
+Please, refer to the `vagrant-libvirt`_ documentation for getting a better understanding of all available options.
+
+There is an example:
+
+.. code-block:: yaml
+
+	---
+	vagrant:
+
+	  platforms:
+	    - name: rhel6
+	      box: rhel/rhel-6
+	    - name: rhel7
+	      box: rhel/rhel-7
+	    - name: centos7
+	      box: centos/7
+
+	  providers:
+	    - name: libvirt
+	      type: libvirt
+	      options:
+		memory: 1024
+		cpus: 2
+		driver: kvm #Note that the two available drivers are kvm and qemu(refer to the vagrant-libvirt doc).
+		video_type: vga
+
+	  instances:
+	    - name: ansible-role
+	      raw_config_args:
+		- "ssh.pty = true"
+		- "vm.synced_folder './', '/vagrant', type: '9p', disabled: true"
+		- "vm.network :private_network, :libvirt__dhcp_enabled=> false ,:libvirt__tunnel_type => 'server', :libvirt__tunnel_port => '11111'"
+	      options:
+		append_platform_to_hostname: no
+
+	      ansible_groups:
+		- group_1
+
+Notes
+^^^^^
+
+* Molecule destroy does not seem to work well. It removes some molecule files but not always destroys de VM. You may have to do it manually by using the virsh command.
```

Comment:
Please raise an [Issue](https://github.com/metacloud/molecule/issue) with details about what you experience when molecule destroy does not work for you.